src/ssl.c

```diff
@@ -17125,39 +17124,14 @@ int wolfSSL_EVP_MD_type(const WOLFSSL_EVP_MD *md)
             case AES_256_GCM_TYPE :
                 WOLFSSL_MSG("AES GCM");
                 if (ctx->enc) {
-                    if (dst){
-                        /* encrypt confidential data*/
-                        ret = wc_AesGcmEncrypt(&ctx->cipher.aes, dst, src, len,
+                    ret = wc_AesGcmEncrypt(&ctx->cipher.aes, dst, src, len,
                                   ctx->iv, ctx->ivSz, ctx->authTag, ctx->authTagSz,
                                   NULL, 0);
-                    }
-                    else {
-                        /* authenticated, non-confidential data */
```

Comment:
This functionality is being removed.  I understand the dst == NULL check above means it can't currently happen.  But is this really the correct change?

Reply:
If the dst == NULL check is removed this code is no longer dead, do you want me to change this fix? (remove the null check and keep this code)